prez/services/listings.py

```diff
@@ -254,44 +242,14 @@ async def ogc_features_listing_function(
         subselect = copy.deepcopy(feature_list_query.inner_select)
         count_query = CountQuery(original_subselect=subselect).to_string()
 
-        # Features listing requires CBD of the Feature Collection as well; reuse items profile to get all props/bns to
-        # depth two.
-        gpnt = GraphPatternNotTriples(
-            content=Bind(
-                expression=Expression.from_primary_expression(
-                    PrimaryExpression(
-                        content=IRIOrFunction(iri=IRI(value=collection_uri))
-                    )
-                ),
-                var=Var(value="focus_node"),
-            )
-        )
-        feature_collection_query = PrezQueryConstructor(
```

Comment:
Is this the removal of the additional standalone query that tries to get details of the FeatureCollection using the Feature-listing template?
If so, add it to the description in the PR.
(I thought it wasn't going to be part of this diff, because I didn't see it mentioned in the description).

Reply:
yes it is - have clarified in PR description now.